projects/etos_suite_runner/src/etos_suite_runner/lib/suite.py

```diff
@@ -37,10 +37,11 @@
 from .log_filter import DuplicateFilter
 
 
-class SubSuite:
+class SubSuite:  # pylint:disable=too-many-instance-attributes
```

Comment:
Do we need this disable? According to pylint docs max should be 7, if I'm not misreading it.

Reply:
We have 8